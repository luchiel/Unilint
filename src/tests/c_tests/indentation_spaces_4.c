void test() {
    //good indent here
}

int main() {
    for(int i = 9; i > 6; --i) {
      //indent too small
        //indent ok
          //indent too deep
    }
    if(1)
        test();
}