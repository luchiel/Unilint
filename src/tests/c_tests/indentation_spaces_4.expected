7:1: indent too small
9:1: indent too deep
